python/tests/torch/test_data.py

```diff
@@ -18,13 +18,18 @@
 
 # Third Party
 import numpy as np
+import pytest
+import torch
 from pyspark.sql import Row, SparkSession
+from torch.utils.data import DataLoader as torchDataLoader
 
 # Rikai
 from rikai.numpy import wrap
-from rikai.torch import DataLoader
+from rikai.torch import DataLoader, Dataset
 from rikai.types import Box2d, Image
 
+torch.multiprocessing.set_sharing_strategy("file_system")
```

Comment:
consider putting this in a session scoped fixture or inside pytest_sessionstart ?

Reply:
I am not very familar with the convention for pytest fixture.

Should this be:
```
@pytest.fixture(scope="session"):
def torch_session():
    torch.multiprocessing.set_sharing_strategy("file_system")
```

and let each of the torch test to declare this parameter?

Reply:
Added a fixture in `python/tests/torch/conftest.py`